e2e_tests/docker_image/Dockerfile

```diff
@@ -102,10 +99,6 @@ ENV PATH="/root/.cargo/bin:${PATH}"
 COPY _exec_wrapper /usr/local/bin/
 RUN ls /opt/rust/bin | xargs -n1 -I% ln -s /usr/local/bin/_exec_wrapper /usr/local/bin/$(basename %)
 
-# Adding the targets for the cross-compilation tests
-RUN rustup target add armv7-unknown-linux-gnueabihf
-RUN rustup target add aarch64-unknown-linux-gnu
-
 # Add users for multitenancy tests
```

Comment:
This `Dockerfile` can be uploaded so that it uses a bit less space 👌

Reply:
You can remove the changes here, I've added them in my PR and the image has already been published! You just need to re-run the CI